src/gazebo_motor_model.cpp

```diff
@@ -188,7 +190,17 @@ void GazeboMotorModel::UpdateForcesAndMoments() {
   // Apply the filter on the motor's velocity.
   double ref_motor_rot_vel;
   ref_motor_rot_vel = rotor_velocity_filter_->updateFilter(ref_motor_rot_vel_, sampling_time_);
+#if 0
   joint_->SetVelocity(0, turning_direction_ * ref_motor_rot_vel / rotor_velocity_slowdown_sim_);
+#elif 0
+  joint_->SetParam("fmax", 0, 2.0);
+  joint_->SetParam("vel", 0, turning_direction_ * ref_motor_rot_vel / rotor_velocity_slowdown_sim_);
+  // gzerr << turning_direction_ * ref_motor_rot_vel / rotor_velocity_slowdown_sim_ << "\n";
+#else
+  double err = joint_->GetVelocity(0) - turning_direction_ * ref_motor_rot_vel / rotor_velocity_slowdown_sim_;
```

Comment:
Is this Gazebo 6 compatible? If not, can this be correctly ifdefed (like in other locations, e.g. gazebo_mavlink_interface.cpp) for Gazebo 6 / 7?